### PR TITLE
Common - Add setting for compass magnetic declination

### PR DESCRIPTION
**Pull request by PabstMirror**
ref map https://www.ncei.noaa.gov/sites/default/files/2022-02/Miller%20Projection%20Main%20Field-%20Annual%20Change%20Declination%20%28D%29.jpg

### CODE REVIEW

File: addons/common/initSettings.inc.sqf [hunk 1]
```diff
@@ -102,6 +102,16 @@ private _categorySway = [_category, LSTRING(subcategory_sway)];
     0
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(magneticDeclination),
+    "CHECKBOX",
+    [LSTRING(magneticDeclination), LSTRING(magneticDeclinationooltip)],
+    _category,
+    false,
+    true,
```
Comment (johnb432) [inline addons/common/initSettings.inc.sqf:111]:
```suggestion
    1,
```

**Review by johnb432 — APPROVED** [review]
Out of curiosity, which year's dataset did you use? I saw 2 PRs trying to update the values to 2020 (https://github.com/cmweiss/geomag/pull/15), would be neat to use those if you don't already use them.

**Comment by PabstMirror** [comment]
IIRC I used that python lib with an updated WMM.COF
I can try to rerun it, and we should see slight changes just going from feb->october

**Review by johnb432 — APPROVED** [review]

**Comment by johnb432** [comment]
Imo push this out for 3.18.2 and we can update the data later if deemed necessary.

**Pull request merged**